infrastructure/resources/policy-files/Hearings/api-op-delete-hearing-policy.xml

```diff
@@ -94,35 +94,8 @@
     <outbound>
         <base />
          <choose>
-            <when condition="@(context.Response.StatusCode == 401)">
+			 <when condition="@(context.Response.StatusCode == 401 || context.Response.StatusCode == 403)">
```

Comment:
Small indentation issue here. Can it be changed?

Reply:
fixed